### ISSUE TITLE
Nginx aliases does not work for hostmanager 

### ISSUE THREAD

**Issue by caseycs** [puphpet/puphpet#1694]
In vagrant environment
```
nginx:
    vhosts:
        nxv_fs9uk0f329v0:
            server_name: xxx
            server_aliases:
                - yyy
                - zzz
```

I'll come up with the PR in few minutes

**Comment by jtreminio** [comment]
Implemented in caf80ee5072dce66c2bbd48e7c56485b51bd3a68

**Comment by jtreminio** [comment]
Working fine as of latest archive:

```
nginx:
    install: '1'
    settings:
        default_vhost: 1
        proxy_buffer_size: 128k
        proxy_buffers: '4 256k'
        proxy_connect_timeout: 600s
        proxy_send_timeout: 600s
        proxy_read_timeout: 600s
        names_hash_bucket_size: 128
    upstreams: {  }
    vhosts:
        nxv_597dajpft4ik:
            server_name: awesome.dev
            server_aliases:
                - www.awesome.dev
                - bam.awesome.dev```

and `/etc/hosts`:

```## vagrant-hostmanager-start id: 9275d55b-1bdf-454b-8b1c-6d922bf33f5a
192.168.56.176  local.puphpet
192.168.56.176  awesome.dev
192.168.56.176  www.awesome.dev
192.168.56.176  bam.awesome.dev
## vagrant-hostmanager-end```
